torchsde/_core/methods/srk.py

```diff
@@ -30,28 +31,28 @@
 from .tableaus import sra1, srid2
 
 
-class BaseSRK(base_solver.BaseSDESolver):
+class SRK(base_solver.BaseSDESolver):
     strong_order = 1.5
     weak_order = 1.5
     sde_type = SDE_TYPES.ito
+    noise_types = (NOISE_TYPES.additive, NOISE_TYPES.diagonal, NOISE_TYPES.scalar)
     levy_area_approximation = LEVY_AREA_APPROXIMATIONS.spacetime
 
-    def __init__(self, sde, bm, y0, dt, adaptive, rtol, atol, dt_min, options, **kwargs):
-        super(BaseSRK, self).__init__(sde=sde, bm=bm, y0=y0, dt=dt, adaptive=adaptive, rtol=rtol, atol=atol,
-                                      dt_min=dt_min, options=options, **kwargs)
+    def __init__(self, sde, options, **kwargs):
+        if sde.noise_type == NOISE_TYPES.additive:
+            self.step = self.additive_step
+        else:
+            self.step = self.diagonal_or_scalar_step
+
+        super(SRK, self).__init__(sde=sde, options=options, **kwargs)
         self.dt1_min = self.options.get('dt1_min', 0.01)
```

Comment:
`dt1_min` and `dt1_div_dt` could be removed.

Reply:
Done.